aire/src/card/dividedcard.ts

```diff
@@ -0,0 +1,30 @@
+import {bindable, customElement, inject}      from "aurelia-framework";
+import {dom}    from "aire/core";
+import {AireCard} from "aire/card/card";
+
+@inject(Element)
+@customElement('aire-divided-card')
+export class AireDividedCard extends AireCard {
+
+  @bindable
+  card        : HTMLElement;
+
+  @bindable
+  title     : string = "";
+
+  @bindable
+  header    : boolean;
+
+  @bindable
+  footer    : boolean;
+
+  constructor(element: Element) {
+    super(element);
+    this.header = element.hasAttribute("header");
```

Comment:
eh, you're doing 2 things here: creating bindable attributes and relying on the actual custom element attributes.  The convention I've been following is 
1. minimize bindables at all cost as they pollute the API
2. use attributes for all that optional crap.